pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -481,6 +481,26 @@ func (m *vmMutator) addNodeSelectorTerms(defaultNamespace string, nodeSelector *
 		}
 	}
 
+	if isDedicatedCPU(vm) {
+		requirement := v1.NodeSelectorRequirement{
+			Key:      kubevirtv1.CPUManager,
+			Operator: v1.NodeSelectorOpExists,
```

Comment:
I have one question: Why this is `Exists` instead of `in (true)` as you mentioned in https://github.com/harvester/harvester/pull/7559#discussion_r1950677387? We can enable cpumanager then disable it, then the cpumanager label could be false, right?

Reply:
I'm also a little undecided here. But I haven't found any clear documentation that it always has to be 'true'. You could also use `yes` or `1` as the value. I have found some code where it checks for `true`, but as long as I have not seen any Kubernetes documentation where it is explicitly specified that boolean values are always mapped with the string `true` I am skeptical.

Reply:
> You could also use yes or 1 as the value.

What I know is the `cpumanager` label is introduced by kubevirt, from the code
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/heartbeat/heartbeat.go#L145, looks like it convert a bool value to string, so it should be either "true" or "false" I thought.

Reply:
I've adapted the code. It is now using the `NodeSelectorOpIn` operator.